src/core/load-fragment.c

```diff
@@ -6573,3 +6575,49 @@ int config_parse_tty_size(
 
         return config_parse_unsigned(unit, filename, line, section, section_line, lvalue, ltype, rvalue, data, userdata);
 }
+
+int config_parse_regex(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+#if HAVE_PCRE2
+        _cleanup_(sym_pcre2_code_freep) pcre2_code *cs = NULL;
+        char **regex = data;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        r = dlopen_pcre2();
+        if (r < 0)
+                return r;
```

Comment:
this failure should be handled gracefully, i.e. not be propagated, but we should just do log_syntax() here, and say that we ignore the config line because we couldn't load pcre

Reply:
moreover, this should be moved after the next check, just in case.